charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
-        {{ if .Values.controllerManager.apiserverSkipVerify -}}
+        {{ if and (.Values.controllerManager.apiserverSkipVerify) (not .Values.useAggregator) -}}
```

Comment:
@MHBauer I see two (similar) alternatives of how to make it more explicit on the user side. If the user explicitly passes the flag, I expect it to be passed "as is" to the service.

**Option 1.** Override the `apiserverSkipVerify` flag in the command line (as part of the [api-aggregation-setup.md#install-the-service-catalog-chart-with-helm](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/api-aggregation-setup.md#install-the-service-catalog-chart-with-helm) instruction):
```
helm install ... --set controllerManager.apiserverSkipVerify=false ...
```
**Option 2.** Make the `apiserverSkipVerify` flag `false` by default in [values.yaml#L99](https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/values.yaml#L99) and ask user to explicitly override it when needed (and do the same in our integration tests). 

**Option 3?** Can we add some extra parameters (CA) to actually be able to validate TLS without ever disabling it?